packages/components/src/toolbar/toolbar/style.scss

```diff
@@ -10,6 +10,7 @@
 
 	&.is-unstyled {
 		border: none;
+		background: transparent;
```

Comment:
The white background doesn't originally exist in the `Toolbar` component itself. It exists in the `BlockToolbar` component, a block that extends the `Toolbar` component.

https://github.com/WordPress/gutenberg/blob/14f9a1c1bfce1f02130150bf8b9d5454cca32dc8/packages/block-editor/src/components/block-toolbar/style.scss#L66

And I think the white background only needs to be overridden when this block toolbar is rendered in the header, so I think adding it in the place suggested in [this comment](https://github.com/WordPress/gutenberg/pull/62225#issuecomment-2145021276) might be a good idea.

Reply:
Thank you for your patience. Moved it there, and it still seems to work for me.